core/components/molecules/sidesheet/__stories__/variants/LayeringWithModal.story.tsx

```diff
@@ -0,0 +1,199 @@
+              Primary
+            </Button>
+          </>
+        }
+      >
+        <Text>Modal Part Two Body</Text>
+        <ModalDescription description="Card Sections include supporting text like an article summary or a restaurant description." />
+      </Modal>
+    </div>
+  );
+};
+
+const customCode = `() => {
+  const [open, setOpen] = React.useState(true);
+  const [openSecond, setOpenSecond] = React.useState(false);
+
+  const onClose = () => {
+    setOpen(!open);
+  };
+
+  const onSecondOverlayClose = () => {
+    setOpenSecond(!openSecond);
+  }
+
+  const headerOptions = {
+    heading: 'Heading',
+    subHeading: 'Subheading'
+  };
+
+  const options = {
+    onClose,
+    open,
+    headerOptions,
+    footer: (
+      <>
+        <Button appearance="primary" className="mr-4" onClick={() => setOpenSecond(true)}>Open</Button>
+        <Button appearance="basic">Basic</Button>
+      </>
+    )
+  };
+
+  const modalDescriptionOptions = {
+    title: 'Description Title',
+    description: 'Adding a subheading clearly indicates the hierarchy of the information.',
+    removePadding: true
+  };
+
+  const modalDescriptionOptionsWithoutTitle = {
+    description: 'Card Sections include supporting text like an article summary or a restaurant description.',
+    removePadding: true
+  };
+
+  return (
+    <div>
+      <Paragraph>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. <br />
+        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. <br />
+        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. <br />
+        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. <br />
+        <Button appearance="primary" onClick={() => setOpen(true)}>Open</Button>
+      </Paragraph>
+      <Sidesheet {...options} >
+        <Text>Modal Body</Text>
+        <ModalDescription {...modalDescriptionOptions} />
+        <ModalDescription {...modalDescriptionOptionsWithoutTitle} />
+      </Sidesheet>
+      <Modal
+        open={openSecond}
+        dimension='medium'
+        backdropClose={false}
+        onClose={onSecondOverlayClose}
+        headerOptions={{
+          heading: 'Heading Part Two',
+          subHeading: 'Subheading Part Two',
+        }}
+        footer={
+          <>
+            <Button appearance="primary" className="ml-4" onClick={() => console.log('Primary button click')}>
+              Primary
+            </Button>
+          </>
+        }
+      >
+        <Text>Modal Part Two Body</Text>
+        <ModalDescription description="Card Sections include supporting text like an article summary or a restaurant description." />
+      </Modal>
+    </div>
+  );
+}`;
+
+export default {
+  title: 'Components/Sidesheet/LayeringWithModal',
```

Comment:
Title should be `Components/Sidesheet/Layering With Modal`

Reply:
Updated.